Lib/test/test_ctypes/test_memfunctions.py

```diff
@@ -77,6 +78,25 @@ def test_wstring_at(self):
         self.assertEqual(wstring_at(a, 16), "Hello, World\0\0\0\0")
         self.assertEqual(wstring_at(a, 0), "")
 
+    def test_memoryview_at(self):
+        b = (c_byte * 10)()
+
+        foreign_ptr = cast(b, c_void_p)
+        foreign_ptr_size = len(b)
+
+        # memoryview_at() is normally used with pointers given to us
+        # by C APIs. It's an efficient way to get a buffer
+        # representing a dynamically-sized memory region without having
+        # to create an array type first.
+        v = memoryview_at(foreign_ptr, foreign_ptr_size)
```

Comment:
Shouldn't the first argument be an *address*, rather than a ctypes pointer?

Reply:
It can be either just like with `string_at()`. It's intended to be used with `c_void_p` objects.

Reply:
Could you add it to the test and docs?

I know the existing tests and docs are lacking, but let's do better today.

Reply:
I'm sorry but I no longer have time to work on this patch. Sorry for wasting your time